### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Kozubowicz**
[DEMO LINK](https://Kozubowicz.github.io/react_people-table-advanced/)

### CODE REVIEW

File: src/components/PeoplePage.tsx [hunk 1]
```diff
@@ -1,35 +1,154 @@
...
+/*
+
+    <p data-cy="peopleLoadingError">Something went wrong</p>
+
+    <p data-cy="noPeopleMessage">
+      There are no people on the server
+    </p>
+
+    <p>There are no people matching the current search criteria</p>
+*/
```
Comment (volodymyr-soltys97) [inline src/components/PeoplePage.tsx:154]: Remove comments
```suggestion
```

File: src/components/PeoplePage.tsx [hunk 2]
```diff
@@ -1,35 +1,154 @@
...
+        peopleCopy = peopleCopy.filter((person) => person.sex === sex);
+      }
+
+      if (selectedCenturies.length > 0) {
+        peopleCopy = peopleCopy.filter((person) => selectedCenturies.includes(
+          (Math.ceil(person.born / 100)).toString(),
+        ));
+      }
+
+      if (sort === 'name' || sort === 'sex') {
```
Comment (volodymyr-soltys97) [inline src/components/PeoplePage.tsx:56]: Will be better if you create a enum for these words and use it here

File: src/components/PeoplePage.tsx [hunk 3]
```diff
@@ -1,35 +1,154 @@
...
+        if (order) {
+          peopleCopy = peopleCopy.sort((person, person2) => (
+            person2[sort] - person[sort]
+          ));
+        } else {
+          peopleCopy = peopleCopy.sort((person, person2) => (
+            person[sort] - person2[sort]
+          ));
+        }
+      }
```
Comment (volodymyr-soltys97) [inline src/components/PeoplePage.tsx:78]: Try to use switch case instead of it/else if for better readability

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Great work 🔥 
Let's improve your code

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome work!

**Pull request closed without merging**